test/integration/integration_suite_test.go

```diff
@@ -117,6 +128,29 @@ var _ = BeforeSuite(func(done Done) {
 		Expect(err).ToNot(HaveOccurred())
 		Expect(c).ToNot(BeNil())
 		k8sClients[cluster] = c
+
+		if withAgent {
+			kubeCtl.SetContext(cluster)
+			err := c.Get(context.Background(), client.ObjectKey{Name: staticVMNS.Name}, &v1.Namespace{})
+			if apierrors.IsNotFound(err) {
+				err = c.Create(context.TODO(), staticVMNS)
+				Expect(err).ToNot(HaveOccurred(), "create static vm namespace")
+			} else {
+				Expect(err).ToNot(HaveOccurred(), "unable to get vm namespace")
+			}
+
+			cmd := exec.Command("../../hack/install-agent-prerequisites.sh")
+			bytes, err := cmd.CombinedOutput()
+			Expect(err).ToNot(HaveOccurred(), "generate antrea agent kubeconfigs", string(bytes))
+			path, err := filepath.Abs("../../hack/install-wrapper.sh")
+			Expect(err).ToNot(HaveOccurred())
+			dir, err := os.Getwd()
+			Expect(err).ToNot(HaveOccurred())
+			_ = os.Setenv("TF_VAR_with_agent", "true")
+			_ = os.Setenv("TF_VAR_vm_agent_k8s_conf", dir+"/antrea-agent.kubeconfig")
```

Comment:
Dont do it in this fashion. Pass location of antrea-agent.kubeconfig to the script by creating a temp dir..